backend/pretix/__init__.py

```diff
@@ -347,3 +344,18 @@ def create_order(conference: Conference, order_data: CreateOrderInput) -> Order:
     data = response.json()
 
     return Order(code=data["code"], payment_url=data["payments"][0]["payment_url"])
+
+
+def get_user_tickets(conference: Conference, email: str):
+    response = pretix(
+        conference=conference,
+        endpoint="tickets/attendee-tickets",
+        method="get",
+        qs={
+            "attendee_email": email,
```

Comment:
does the query string appear in the logs? I am worried it might cause GDPR issues

Reply:
cloudwatch logs get deleted every 2 weeks so it might be fine for now!